test/quantization/test_quantize_script.py

```diff
@@ -695,6 +695,43 @@ def forward(self, x):
         m = prepare_script(m, {'': default_qconfig})
         assert len(attrs_with_prefix(m, '_observer_',)) == 3
 
+    def test_insert_observers_for_nested_if(self):
+        class Res(torch.nn.Module):
+            def __init__(self, use_skip):
+                super(Res, self).__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 1).float()
+                self.cond = use_skip
+                self.use_skip = use_skip
+
+            def forward(self, x):
+                if self.use_skip:
+                    if self.cond:
+                        return self.conv(x)
```

Comment:
Should we test with different ops in each conditional statement? Can help increase the coverage of this test.

Reply:
Yeah, I think we can test different types of ops here, but maybe we can remove "nested" and just test `if`, since there is actually not much difference between `if` and `nested if`. I'll add one more test for a different type